test-tools/wamr-ide/VSCode-Extension/resource/scripts/boot_debugger_server.sh

```diff
@@ -1,13 +1,13 @@
+#!/bin/bash
+
 # Copyright (C) 2019 Intel Corporation.  All rights reserved.
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
-#!/bin/bash
-
 docker run -it --name=wasm-debug-server-ctr \
            -v "$(pwd)":/mnt \
            -p 1234:1234 \
            wasm-debug-server:1.0 \
            /bin/bash -c "./debug.sh $1"
 
 docker stop wasm-debug-server-ctr>/dev/null
-docker rm wasm-debug-server-ctr>/dev/null
\ No newline at end of file
+docker rm wasm-debug-server-ctr>/dev/null
```

Comment:
Shall we use `docker rm -f` like the command modified in other files?

Reply:
i simplified them differently.